src/commcare_cloud/ansible/roles/commcare-audit/tasks/main.yml

```diff
@@ -1,38 +1,21 @@
 ---
-- name: Check if /opt/data exists
-  stat:
-    path: /opt/data
-  register: data_dir
+- name: Set host's audit directory fact
```

Comment:
nit: I guess this could just be "Set host's audit directory"? 
Is `host_audit_directory` used elsewhere as well? I see only one usage that can have this set directly.